deploy/charts/cert-manager/templates/_helpers.tpl

```diff
@@ -23,13 +23,6 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
-{{/*
-Create chart name and version as used by the chart label.
-*/}}
-{{- define "cert-manager.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-
```

Comment:
I wonder why we used to define a `.chart` variable for each component, if they all had the same value?

Reply:
Not sure, but https://github.com/jetstack/cert-manager/commit/4586ecf9309c5a73edabecf3caf5ecd206e67ef4 shows that this code was part of another Chart.